### ISSUE TITLE
Ingress > WAF > Don't create new stack

### ISSUE THREAD

**Issue by janavenkat** [zalando-incubator/kube-ingress-aws-controller#446]
WAF is already provisioned by client side. So that adding the parameter `--aws-waf-web-acl-id` controller should not create the new cloud formation/load-balancer stack or is there any restriction? that's why controller is creating new stack.

controller version: v0.12.5

**Comment by szuecs** [comment]
@janavenkat sounds like a maybe bug. Does it hurt?

**Comment by janavenkat** [comment]
@szuecs yeah am wondering we could do the association without create a new stack

**Comment by janavenkat** [comment]
> @janavenkat sounds like a maybe bug. Does it hurt?

It's hard to say it's a bug, it does the work the only problem i see it creates new stack. can we have this in optimization list?
